contracts/contracts/utils/AerodromeAMOQuoter.sol

```diff
@@ -384,10 +381,13 @@ contract QuoterHelper {
             );
 
             if (
-                low == high ||
                 isWithinAllowedVariance(sqrtPriceX96After, sqrtPriceTargetX96)
             ) {
                 return (mid, iterations, swapWETHForOETHB, sqrtPriceX96After);
+            } else if (low == high) {
+                // target swap amount not found.
+                // try increasing BINARY_MAX_AMOUNT_FOR_PUSH_PRICE
+                revert("SwapAmountNotFound");
```

Comment:
Indeed, nice catch! I now understand why I sometimes had false positives. I thought the issue came from somewhere else, thank you for pointing that out!